simapp/app.go

```diff
@@ -97,7 +97,7 @@ var (
 	// and genesis verification.
 	ModuleBasics = module.NewBasicManager(
 		auth.AppModuleBasic{},
-		genutil.AppModuleBasic{},
+		genutil.NewAppModuleBasic(genutiltypes.DefaultMessageValidator),
```

Comment:
Don't we need to add the same in `app_legacy.go`?

Reply:
added